pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -18,6 +18,11 @@ import (
 
 const priorityClassKey = "priorityClassName"
 
+const (
+	fleetChartValuesConfigMapNamespace = "cattle-system"
```

Comment:
:thinking: why do we need these constants?

Reply:
You are right, they are leftovers from my initial approach. Thanks!